sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Security/Ucfg/UcfgBuilder_Instructions.cs

```diff
@@ -491,6 +497,223 @@ public interface IBar<T>
             UcfgVerifier.VerifyInstructions(code, "Foobar");
         }
 
+        [TestMethod]
+        public void Invocations_MethodsWithDefaultParameters()
+        {
+            const string code = @"
+namespace Namespace
+{
+    public class Class1
+    {
+        public void Foo()
+        {
+            SendEmailAsync(""body"");                   // %0 := Namespace.Class1.SendEmailAsync(string, bool) [ this const ]
+            SendEmailAsync(""body"", isHtml: true);     // %1 := Namespace.Class1.SendEmailAsync(string, bool) [ this const const ]
+            SendEmailAsync(""body"", isHtml: false);    // %2 := Namespace.Class1.SendEmailAsync(string, bool) [ this const const ]
+        }
+
+        public System.Threading.Tasks.Task SendEmailAsync(string body, bool isHtml = false)
+        { return null; }
+    }
+}";
+            UcfgVerifier.VerifyInstructions(code, "Foo");
+        }
+
+        [TestMethod]
+        public void Invocations_UsedNamedParameters()
+        {
+            const string code = @"
+namespace Namespace
+{
+    public class Class1
+    {
+        public void Foo()
+        {
+            SendEmailAsync(""data"", false);                  // %0 := Namespace.Class1.SendEmailAsync(string, bool) [ this const const ]
+            SendEmailAsync(body: ""data"", isHtml: false);    // %1 := Namespace.Class1.SendEmailAsync(string, bool) [ this const const ]
+            SendEmailAsync(body: ""data"");                   // %2 := Namespace.Class1.SendEmailAsync(string, bool) [ this const ]
+            SendEmailAsync(isHtml: false, body: ""data"");    // %3 := Namespace.Class1.SendEmailAsync(string, bool) [ this const const ]
+        }
+
+        public System.Threading.Tasks.Task SendEmailAsync(string body, bool isHtml = false)
+        { return null; }
+    }
+}";
+            UcfgVerifier.VerifyInstructions(code, "Foo");
+        }
+
+        [TestMethod]
+        public void Invocations_WIP()

```

Comment:
Is it still needed? If so please rename the test.

Reply:
Not needed, along with some of the other "Simple" tests I added. I'll remove them.